packages/hooks/useIpfs.ts

```diff
@@ -22,6 +24,7 @@ interface IPFSUploadProgress {
 export interface PinataFileProps {
   file: LocalFileData;
   pinataJWTKey: string;
+  mode?: AppMode;
```

Comment:
I don't understand why this hook need to be aware of the app mode

Reply:
While uploading file from app instead of converting selected file to  File type similar to WEB, I just used file URL to create formData and upload it to pinata. So, just to check if we are uploading from web or app I added that appmode

Reply:
okay, app mode is not related to mobile or desktop, it's the mode of the ui, you could use mini-mode on desktop, btw mini means minimalist, not small screen

Reply:
use Platform.OS in here directly if possible and remove the new prop

Reply:
ok, I didn't knew that. Will do

Reply:
I replaced app mode check with Platform.OS check